src/ldc/eh/msvc.d

```diff
@@ -377,14 +383,14 @@ void msvc_eh_terminate() nothrow
             mov RBX, 0xFFFFFF;
             and RDX, RBX;
             cmp RDX, 0xC48348;          // add ESP,nn  (debug UCRT libs)
-            je L_addESP_found;
+            je L_addESP_found;

```

Comment:
I just edit the file on the github (using the chrome browser), I really do not know why the line endings changed?